### PR TITLE
update iasm instruction table for dlang/dmd#6364

### PR DESCRIPTION
**Pull request by BBasile**
Also adds `popcnt` that missed.

### CODE REVIEW

File: spec/iasm.dd [hunk 1]
```diff
@@ -1067,6 +1067,16 @@ $(H3 $(LNAME2 amd_opcode_support, AMD Opcodes Supported))
         )
     )
 
+$(H3 $(LNAME2 Bit Manipulation Instructions, Bit manipulation instructions supported))
```
Comment (Hackerpilot) [inline spec/iasm.dd]: Why do these instructions have their own section?

Comment (BBasile) [inline spec/iasm.dd]: three reasons:
- the big table is sorted alphabetically so it's unpleasant to insert something at the middle.
- these instructions are almost part of a special set called ABM1 and ABM2 (Advanced Bit Manipulation) which are, despite of not being specific to vectors, SSE2 only.
- I may propose more iasm ABM in a further PR.

Comment (BBasile) [inline spec/iasm.dd]: And because of the 3rd point This PR doesn't need to be merged right now !

**Comment by wilzbach** [comment]
>> I may propose more iasm ABM in a further PR.
> And because of the 3rd point This PR doesn't need to be merged right now !

@BBasile could you please only close spec update PRs if you have replacements for them?
It's really easy to loose track of them otherwise.

(it seems that you have force-pushed this branch, so I can't reopen it)

**Comment by BBasile** [comment]
Yes I didn't remember that I made the error to commit the change in master. I've forced the push of my fork to keep it up to date and closes against my will this PR. I'd say that here it doesn't matter as I might propose more later. The table with just one entry was ugly anyway.

**Pull request closed without merging**